kratos/python/add_processes_to_python.cpp

```diff
@@ -238,8 +238,12 @@ void  AddProcessesToPython(pybind11::module& m)
     ;
 
     py::class_<FindConditionsNeighboursProcess, FindConditionsNeighboursProcess::Pointer, Process>(m,"FindConditionsNeighboursProcess")
-            .def(py::init<ModelPart&, int, unsigned int>())
-    .def("ClearNeighbours",&FindConditionsNeighboursProcess::ClearNeighbours)
+        .def(py::init<Model&, Parameters>())
+        .def(py::init<ModelPart&>())
+        .def(py::init<ModelPart&, const int>())
```

Comment:
I wouldn't expose these two to Python to promote the use of the Model-Parameters constructor rather than the old ones.

Reply:
Huuum, Okay, but I will preserve old behaviour to avoid breaking code

Reply:
Sure. Lets keep the one that was in there and only add the model-parameters one.